.github/workflows/python_tests.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ "3.8", "3.10" ]
+        python-version: [ "3.8", "3.11" ]
```

Comment:
Now that we are running tests for python 3.8 and 3.11, we now need to update the code cov "if statements" to use one of these python versions instead of 3.10. eg. https://github.com/nucypher/nucypher/pull/3208/files#diff-df62b370065991dee949f7e19f1e99442c52f7023d3a572a4166d1597b7f0c61L55 (and all others) could be:

```
if: matrix.python-version == '3.8'
...
if: matrix.python-version != '3.8'
```

so that code cov would run for the 3.8 tests and not for 3.11 respectively. We could use 3.11 instead of 3.8 for code cov but I figure that may change to 3.12 or higher before we change the min supported version from 3.8.

Also, there may be a way to define a job variable that we can then use for the check, so that changing it in the future is easier and more maintainable - https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow - something like:

```yml
jobs:
   python-tests:
     env:
       CODE_COV_PY_VERSION: 3.8
     steps:
       ...
       - name: Unit Tests (Coverage)
          if: matrix.python-version == "$CODE_COV_PY_VERSION"
            <unit tests with coverage>
       - name: Unit Tests
          if: matrix.python-version != "$CODE_COV_PY_VERSION"
            <unit test with no coverage>
       ...
                         
```

Reply:
✔️ kept the existing approach for now, just bumped the version to run codecov along with 3.11.